pkg/skaffold/build/gcb/types.go

```diff
@@ -120,3 +122,7 @@ func (b *Builder) Prune(ctx context.Context, out io.Writer) error {
 func (b *Builder) PushImages() bool {
 	return true
 }
+
+func (b *Builder) SupportedPlatforms() platform.Matcher {
+	return platform.Matcher{Platforms: []v1.Platform{{OS: "linux", Architecture: "amd64"}}}
```

Comment:
Let's confirm if this is the only supported platform.

Reply:
it isn't. But this is just to block users trying out specifying platforms for GCB builders that aren't yet implemented. Will be removed after implementing it on a builder by builder basis.